dashboard/test/models/level_test.rb

```diff
@@ -586,4 +586,43 @@ def create_maze
     assert_equal data[:description], level.description
     assert_equal data[:description], I18n.t("data.unplugged.#{data[:name]}.desc")
   end
+
+  test "audit log will explode properties" do
+    data = @custom_maze_data.dup
+    data[:name] = "test audit log"
+    level = Level.create(data)
+    assert level.valid?
+    assert_nil level.audit_log
```

Comment:
If these are things needing confirmation, I'd argue that they should be separate tests.